Source/WebKit/NetworkProcess/storage/FileSystemStorageHandle.cpp

```diff
@@ -86,6 +86,8 @@ void FileSystemStorageHandle::close()
 
     if (m_activeSyncAccessHandle)
         closeSyncAccessHandle(m_activeSyncAccessHandle->identifier);
+
+    closeWritable(true);
```

Comment:
It might be nicer to replace the boolean with an enum to make it clear that we are aborting.
Or `bool aborted = true; closeWritable(aborted);`.

Reply:
It's a good idea to make `aborted` enum; I will replace it in all places (including IPC message) in a followup patch.